mono_repo/lib/src/commands/add_overrides.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:path/path.dart' as p;
+import 'package:pubspec_parse/pubspec_parse.dart';
+import 'package:yaml/yaml.dart';
+import 'package:yaml_edit/yaml_edit.dart';
+
+import '../../mono_repo.dart';
+import '../root_config.dart';
+import 'mono_repo_command.dart';
+
+class AddOverridesCommand extends MonoRepoCommand {
```

Comment:
Presumably we also want a `RemoveOverridesCommand` (for publishing?)

Reply:
Likely. Or one should just revert?

Reply:
It is common to leave overrides in though as well (see the test repo). But then you need to remove them before publishing.

A bit of a tangent but now that I think about it this could also be useful for setting dependency constraints more generally as well? We could support a `dependencies` (and `dev_dependencies`) section where you could set the versions of all deps that you want to be consistent across packages.